### PR TITLE
Add option list subcommand.

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Could be merged now?

File: features/option.feature [hunk 1]
```diff
@@ -13,13 +13,50 @@ Feature: Manage WordPress options
       bar
       """
 
+    When I run `wp option list`
+    Then STDOUT should not be empty
+
+    When I run `wp option list`
+    Then STDOUT should contain:
```
Comment (danielbachhuber) [inline features/option.feature]: Can we test for tabular format?

File: features/option.feature [hunk 2]
```diff
@@ -13,13 +13,50 @@ Feature: Manage WordPress options
...
+      """
+      bar
+      """
+
+    When I run `wp option list --autoload=on`
+    Then STDOUT should not be empty
```
Comment (danielbachhuber) [inline features/option.feature]: We should make this test more specific

**Comment by danielbachhuber** [comment]
> OK?

Sorry, must've missed this earlier. Two minor nits and then it's ready to go.

File: features/option.feature (hunk 1 above)
Comment (szepeviktor) [inline features/option.feature]: Is there a Behat `given` *table that includes this row*?
I think there isn't.

**Comment by szepeviktor** [comment]
I've put a `TAB` in `should contain` strings.
@danielbachhuber  It is OK for you?

**Pull request merged**